File: [LFP]Proyecto_Final_201709426/[LFP]Proyecto_Final_201709426/src/lfp/proyecto_final_201709426/Model/Errror.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package lfp.proyecto_final_201709426.Model;

/**
 *
 * @author Samuel
 */
public class Errror extends Token{
    private String descripcion;
    public Errror() {
    }

    public Errror(String descripcion, String lexema, int numero_token, Object valor, int fila, int columna, String tipo) {
        super(lexema, numero_token, valor, fila, columna, tipo);
        this.descripcion = descripcion;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    @Override
    public String toString() {
        return super.toString() + "Descripcion: " + getDescripcion();
    }
    
}

File: [LFP]Practica2_201709426/[LFP]Practica2_201709426/Model/Errror.vb
Public Class Errror
    Dim lexema As String
    Dim fila As Integer
    Dim columna As Integer
    Dim descripcion As String

    Public Sub New(lexema As String, fila As Integer, columna As Integer, descripcion As String)
        Me.lexema = lexema
        Me.fila = fila
        Me.columna = columna
        Me.descripcion = descripcion
    End Sub

    Public Overrides Function ToString() As String
        Return "Token: " + lexema + "   |  " + "Fila: " + CType(fila, String) + "  |   Columna: " + CType(columna, String) + "  |   descripcion: " + descripcion
    End Function
End Class
